dis_snek/models/discord/role.py

```diff
@@ -16,8 +16,10 @@
     from dis_snek.models.discord.user import Member
     from dis_snek.models.discord.snowflake import Snowflake_Type
 
+T = TypeVar("T")
 
-def sentinel_converter(value, sentinel=attr.NOTHING):
+
+def sentinel_converter(value: bool | T, sentinel: T = attr.NOTHING) -> bool:
```

Comment:
I think `value` should be also Optional as it can be None as per code

Reply:
Looking at how attr converter works, a value should always be passed into a converter method.

Reply:
Optional == can be something OR `None`
And well, if this PR type hints this function, it better be more accurate?

Reply:
yea i suppose so

Reply:
Good catch